pkg/plugin/logger.go

```diff
@@ -24,8 +24,9 @@ import (
 
 // NewLogger returns a logger that is suitable for use within an
 // Ark plugin.
-func NewLogger() logrus.FieldLogger {
+func NewLogger(level logrus.Level) logrus.FieldLogger {
```

Comment:
This is a breaking change to plugins and will require an update in the ark-plugin-example repo